hack/local-development/common/local-imagevector-overwrite

```diff
@@ -35,15 +43,15 @@ cleanup_imagevector_overwrite() {
 
 compute_imagevector_overwrite() {
   imagevector_overwrite="images: []"
-
-  VERSION="$(cat $(dirname $0)/../../VERSION)"
+  # expect VERSION file in repository root
+  VERSION="$(cat "$2/VERSION")"
```

Comment:
```suggestion
  VERSION="$(cat "$REPO_ROOT/VERSION")"
```

Reply:
Thanks, did not know that you can use variables declared in another function.